package.json

```diff
@@ -8,17 +8,18 @@
     "test"
   ],
   "scripts": {
-    "test": "scripts/test.sh",
     "lint:js": "eslint .",
     "lint:js:fix": "eslint . --fix",
     "lint:sol": "solium -d .",
     "lint:sol:fix": "solium -d . --fix",
     "lint": "npm run lint:js && npm run lint:sol",
     "lint:fix": "npm run lint:js:fix && npm run lint:sol:fix",
     "console": "truffle console",
+    "build": "scripts/build.sh",
+    "compile": "scripts/compile.sh",
     "coverage": "scripts/coverage.sh",
     "version": "scripts/version.js",
-    "build": "scripts/build.sh",
+    "test": "npm run compile && scripts/test.sh",
```

Comment:
Why this ninja reordering?

Reply:
alphasort!

Reply:
How is `lint` < `coverage`? :man_shrugging: